action/envelope.go

```diff
@@ -139,8 +139,15 @@ func (elp *envelope) LoadProto(pbAct *iotextypes.ActionCore) error {
 	elp.version = pbAct.GetVersion()
 	elp.nonce = pbAct.GetNonce()
 	elp.gasLimit = pbAct.GetGasLimit()
-	elp.gasPrice = &big.Int{}
-	elp.gasPrice.SetString(pbAct.GetGasPrice(), 10)
+	if pbAct.GetGasPrice() == "" {
+		elp.gasPrice = big.NewInt(0)
+	} else {
+		gp, ok := new(big.Int).SetString(pbAct.GetGasPrice(), 10)
```

Comment:
Both `elp.gasPrice = gp` and `elp.gasPrice, ok := new(big.Int).SetString()` are ok. B/c elp.gasPrice is nil initially

Reply:
`elp.gasPrice, ok := new(big.Int).SetString()` can cause `expected identifier on the left side of :=` error.
Since `elp.gasPrice` has already declared, I use gp instead.